src/styles/components/_toast.scss

```diff
@@ -35,6 +35,7 @@ body {
   .Toastify__toast-container {
     z-index: var(--neeto-ui-toastr-z-index);
     padding: 0;
+    bottom: 60px !important;
```

Comment:
@akkshaylawrence _a Let’s revert this change. If we override the Toast container bottom offset to 60px, it will also affect other Toast positions. We can provide a fix from neetoMolecules instead.

cc @ajmaln